semcore/notice/src/NoticeSmart.jsx

```diff
@@ -12,11 +12,13 @@ class NoticeSmart extends Component {
 
   render() {
     const SNoticeSmart = Root;
-    const { Children, label, actions, closable, onClose } = this.asProps;
+    const { Children, label, title, text, actions, closable, onClose } = this.asProps;
+
     return (
       <SNoticeSmart render={Notice}>
         {isNode(label) && <Notice.Label>{label}</Notice.Label>}
         <Notice.Content>
+          {isNode(title) && <Notice.Title>{title}</Notice.Title>}
           <Children />
```

Comment:
I think, `Children` should be wrapped into `Notice.Text`

Reply:
I'm thinking how to not mess up styles of users' notices.

What do you think about adding here a check: if not string then wrap with ` Notice.Text`, otherwise render `Children` as is?

Reply:
Decided to add check and todo with to remove check in some next major

Reply:
Done